Hi3Helper.EncTool.Test/Program.cs

```diff
@@ -7,10 +7,10 @@ internal class Program
     {
         static async Task Main(string[] args)
         {
-            using (SRMetadata srm = new SRMetadata("https://globaldp-prod-os01.starrails.com/query_dispatch", "495e8edf66",
+            using (SRMetadata srm = new SRMetadata("https://globaldp-prod-os01.starrails.com/query_dispatch", "3a57430d8d",
```

Comment:
Genshin EncTool testing :modcheck:

Reply:
@neon-nyan is trying to figure it out since code reference used by some repair methods are scattered all over the place. Especially for Genshin as its scattered on 3 projects, main app CollapseLauncher, Core, and EncTool.
This will require moving all the used methods for Genshin repair to EncTool which complicates the submodules and the implementation of it back to the main app. 

So this is a bigger project by itself and require more time that it needs for just an internal testing tool. As of right now, it's fine as is as we hardly ever uses the tester for GI or HI3.

Reply:
That's fair, I believe that we can work on the implementation for Hi3 once 2.0 is closer to release.